### PR TITLE
Add build script for airline [JAR]

### PR DESCRIPTION
**Pull request by vedang-wartikar**
[INFO] Tests run: 99, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.875 s - in TestSuite
[INFO]
[INFO] Results:
[INFO]
[INFO] Tests run: 99, Failures: 0, Errors: 0, Skipped: 0

### CODE REVIEW

**Comment by vedang-wartikar** [comment]
Add build script for JAR package - activation. Committed the changes on my forked repo, tried creating a different PR for this, but the changes got reflected on this PR

File: a/airline/airline_rhel7.6.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# setup java environment
+yum install java java-devel -y
+JAVA_HOME="/usr/lib/jvm/java-1.8.0-openjdk"
+echo $JAVA_HOME
+export PATH=$PATH:$JAVA_HOME/bin
+java -version
+
+# install maven
+dnf install maven -y
```
Comment (seth-priya) [inline a/airline/airline_rhel7.6.sh:32]: why are we mixing yum and dnf commands? which base was this tested on? RHEL/UBI/UBI minimal?

**Comment by seth-priya** [comment]
Please squash the commits, @vedang-wartikar

File: a/airline/airline_rhel7.6.sh (hunk 1 above)
Comment (vedang-wartikar) [reply a/airline/airline_rhel7.6.sh:32]: this was initially tested on a centos container. didn't have a RHEL VM when I was creating this build script

**Pull request merged**